src/commands/utils/extensions.ts

```diff
@@ -0,0 +1,17 @@
+import * as vscode from 'vscode';
+
+export function checkExtension(ext: string): boolean {
```

Comment:
💡 Note to self, I will be renaming this file to like "aiextensionnandler.ts` or something closed.

Reply:
good point I initially got confused and then realized it is utils/extensions.ts :)

Reply:
Suggestion: it could be something simpler like utils/handlers.ts ? As these would be util functions for GHCP handlers.

Reply:
If this will be reused may be we can name it as utils/extensionhandlers.ts or ghCopilotHandlers.ts ? something which is not similar to existing files.

Reply:
This was renamed